keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/DeliveryConfig.kt

```diff
@@ -12,7 +12,7 @@ data class DeliveryConfig(
   val serviceAccount: String,
   val artifacts: Set<DeliveryArtifact> = emptySet(),
   val environments: Set<Environment> = emptySet(),
-  val apiVersion: ApiVersion = ApiVersion("delivery.config.spinnaker.netflix.com", "v1")
```

Comment:
Question - why were `ApiVersion` and `ResoucreId` objects and not Strings from the first place?

Reply:
The intention was to restrict the format really, and to not confuse them with other values